test_profile/config/sync/core.entity_form_display.node.external_link.default.yml

```diff
@@ -110,4 +113,5 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
-hidden: {  }
+hidden:
+  field_top_level_theme: true
```

Comment:
top level theme is removed further up and then set to hidden?

Reply:
think it is just moved from 'visible' to 'hidden'

Reply:
Is the theme taxo no longer a requirement?